src/navigations/HomeBottomTabs.js

```diff
@@ -339,15 +340,16 @@ function HomeBottomTabs({navigation}) {
         />
       </Tab.Navigator>
       <FirebaseConfig navigation={navigation} />
-    </View>
+    </SafeAreaView>
   );
 }
 
 export default HomeBottomTabs;
 const styles = StyleSheet.create({
   container: {
     height: '100%',
-    width: '100%'
+    width: '100%',
+    backgroundColor: 'white'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `backgroundColor: 'white'` to the container style is a good practice, as it ensures that the background color is explicitly set and doesn't rely on default styles which may vary across different platforms or versions.

```diff
  const styles = StyleSheet.create({
    container: {
      height: '100%',
      width: '100%',
+     backgroundColor: 'white'
    },
    badge: {
      height: 7,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@asaadam background color is still not using `utils/theme` colors